db/default_messages/upcoming_checkout_email

```diff
@@ -1,4 +1,7 @@
-  Hey there, you have equipment reserved today! Please checkout the following items today:\n\n\
-  @equipment_list@\n\n\
-  If you fail to return your equipment on time you will be subject to a late fee of @late_fee@ per day. If you have lost the item you may additionally have to pay a replacement fee of @replacement_fee@.\n\
-  Log in to Reservations to see if any of your items are eligible for renewal. If you have further questions feel free to contact an employee of @department_name@.\n\n\
+Hey there, you have equipment reserved today! Please checkout the following items today:
+
+@equipment_list@
+
+If you fail to return your equipment on time you will be subject to a late fee of @late_fee@ per day. If you have lost the item you may additionally have to pay a replacement fee of @replacement_fee@.
+Log in to Reservations to see if any of your items are eligible for renewal. If you have further questions feel free to contact an employee of @department_name@.
```

Comment:
I feel like we could use an extra line break here and [above](https://github.com/YaleSTC/reservations/pull/1264/files#diff-f21bad6ec063242ba5b73ea6f53ca870R6).